scripts/ImportExportTest.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Would this script work with these?
```suggestion
set -euo pipefail
```

I wanted to do that last time I was going through these scripts but I did not have time to test each one individually and make sure it does not break anything. Now that you're rewriting it, it looks like a good opportunity to try this.

Reply:
Sadly its not just working. But you're right, it would make sense to do this. I will look into this tomorrow.

Reply:
Ok, I did this in https://github.com/ethereum/solidity/pull/12834/commits/a36a1a2c2dbea4ebfdb7c14c23e46276c1d43297. But somehow it feels like a hack.